### PR TITLE
Swashbuckle article improvements

### PR DESCRIPTION
**Pull request by Rafase282**
Improvements requested on #692

### CODE REVIEW

**Comment by theflametrooper** [comment]
## LGTM :+1:

File: Swashbuckle-Swagger-Operation-Alphabetical-Order.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Arranging operations alphabetically in Swashbuckle Swagger
 
-[Swashbuckle](https://github.com/domaindrivendev/Swashbuckle) seamlessly adds a [Swagger](http://swagger.io/) to WebAPI projects. However, the operations on the page do not appear in alphabetical order. However, Swashbuckle allows custom filters and with a little code, this is easy enough to implement. The code below creates a [DocumentFilter](https://github.com/domaindrivendev/Swashbuckle#documentfilter) which is then applied to the final document as outlined [here](https://github.com/domaindrivendev/Swashbuckle#modifying-generated-operations).
+[Swashbuckle](https://github.com/domaindrivendev/Swashbuckle) seamlessly adds a [Swagger](http://swagger.io/) to WebAPI projects. However, the operations on the page do not appear in alphabetical order.
 
-The Gist is available [here](https://gist.github.com/pallu/0f28e98fa89d2855a321).
+Although the operations will not appear in alphabetical order, Swashbuckle allows custom filters and with a little coding, the order can be shown in alphabetical order.
```
Comment (abhisekp) [inline Swashbuckle-Swagger-Operation-Alphabetical-Order.md]: Add a heading 2 

```md
## (put a space after ##)
```

Comment (Rafase282) [inline Swashbuckle-Swagger-Operation-Alphabetical-Order.md]: What exactly do you want me to turn into a heading 2?

Comment (abhisekp) [inline Swashbuckle-Swagger-Operation-Alphabetical-Order.md]: @Rafase282 Nothing. Just a blank header 2 for your luv <3 towards @camperbot 

But in this case, it was nice to make the **Code Example** header 2. :+1:

Comment (Rafase282) [inline Swashbuckle-Swagger-Operation-Alphabetical-Order.md]: I added a heading 2, the empty one looked ugly on the preview so I just made `## Code Example`

Let me know if that is acceptable.

Comment (abhisekp) [inline Swashbuckle-Swagger-Operation-Alphabetical-Order.md]: yep! It's good :+1: :+1:

**Pull request merged**